electrum/gui/qt/__init__.py

```diff
@@ -53,11 +53,18 @@
 
 
 try:
-    from . import icons_rc
+    import importlib
+    from pathlib import Path
+    path = str(Path(__file__).parent.joinpath("icons/icons_rc.py"))
+    spec = importlib.util.spec_from_file_location("icons_rc", path)
+    module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(module)
 except Exception as e:
     print(e)
     print("Error: Could not find icons file.")
-    print("Please run 'pyrcc5 icons.qrc -o electrum/gui/qt/icons_rc.py'")
+    print("If you're running electrum from the repo then please make sure that")
+    print("the submdules are initialized.")
+    print("git submodule init")
```

Comment:
I think `git submodule update` is necessary in this situation too.

Reply:
I will change it to `git submodule update --init`.